### PR TITLE
Update appxmanifest.xml inproc servers snippet

### PR DESCRIPTION
**Pull request by DrusTheAxe**
Updated the appxmanifest.xml snippet fed to the build pipeline to include in the final PRfwk's appxmanifest.xml

### CODE REVIEW

File: build/AzurePipelinesTemplates/ProjectReunion-PublishProjectOutput-Steps.yml [hunk 1]
```diff
@@ -9,6 +9,18 @@ steps:
       filePath: build\CopyFilesToStagingDir.ps1
       arguments: -BuildOutputDir '$(buildOutputDir)' -PublishDir '$(Build.ArtifactStagingDirectory)\${{ parameters.artifactName }}' -NugetDir '$(Build.ArtifactStagingDirectory)\FullNuget' -Platform '$(buildPlatform)' -Configuration '$(buildConfiguration)'
 
+  - script: |
```
Comment (BenJKuhn) [inline build/AzurePipelinesTemplates/ProjectReunion-PublishProjectOutput-Steps.yml:12]: are these debugging output that you had intended to remove?

**Review by BenJKuhn — APPROVED** [review]
consider removing logging code if unnecessary, otherwise change the task names to something descriptive, rather than your initials.

File: build/AzurePipelinesTemplates/ProjectReunion-PublishProjectOutput-Steps.yml (hunk 1 above)
Comment (DrusTheAxe) [reply build/AzurePipelinesTemplates/ProjectReunion-PublishProjectOutput-Steps.yml:12]: Yep. Gone in latest

**Pull request merged**